data_safe_haven/infrastructure/components/composite/nfsv3_storage_account.py

```diff
@@ -54,21 +51,17 @@ def __init__(
         child_opts = ResourceOptions.merge(opts, ResourceOptions(parent=self))
         child_tags = {"component": "data"} | (tags if tags else {})
 
-        if props.allowed_service_tag == AzureServiceTag.INTERNET:
-            default_action = storage.DefaultAction.ALLOW
-            ip_rules = []
-        else:
-            default_action = storage.DefaultAction.DENY
-            ip_rules = Output.from_input(props.allowed_ip_addresses).apply(
-                lambda ip_ranges: [
-                    storage.IPRuleArgs(
-                        action=storage.Action.ALLOW,
-                        i_p_address_or_range=str(ip_address),
-                    )
-                    for ip_range in sorted(ip_ranges)
-                    for ip_address in AzureIPv4Range.from_cidr(ip_range).all_ips()
-                ]
-            )
+        default_action = storage.DefaultAction.DENY
+        ip_rules = Output.from_input(props.allowed_ip_addresses).apply(
+            lambda ip_ranges: [
+                storage.IPRuleArgs(
+                    action=storage.Action.ALLOW,
+                    i_p_address_or_range=str(ip_address),
+                )
+                for ip_range in sorted(ip_ranges)
+                for ip_address in AzureIPv4Range.from_cidr(ip_range).all_ips()
+            ]
+        )
```

Comment:
I don't quite follow which bit we don't need; the default action?

Reply:
Sorry, I just mean it is a bit redundant to define a variable which you only use in one place like,

```python
default_action = DENY
StorageAccount(
    default_action=default_action,
)
```

instead of 

```python
StorageAccount(
    default_action=DENY,
)
```